### PR TITLE
[DOCS-6813] Support for Opensearch

### PR DESCRIPTION
**Pull request by kcichonczyk**

### CODE REVIEW

**Comment by oliveralfresco2018** [comment]
@mjstanford for the Opensearch inclusion in the 3.2 documentation are you happy with the only changes I make being what is included in this PR ?
cheers

File: search-enterprise/latest/support/index.md [hunk 1]
```diff
@@ -11,7 +11,8 @@ The following are the supported platforms for Search Enterprise 3.1.x:
 | Content Services 7.2.x | Including Alfresco ActiveMQ 5.15.8, Alfresco Transform Service 1.4.1 and PostgreSQL 11.7 |
 | Content Services 7.1.x | Including Alfresco ActiveMQ 5.15.8, Alfresco Transform Service 1.4.1 and PostgreSQL 11.7 |
 | JDK 11 or OpenJDK 11 | |
-| Elasticsearch server 7.10.1 | |
-| Alfresco Elasticsearch Connector 3.1.x | |
+| Elasticsearch server 7.17.3 | |
```
Comment (mjstanford) [inline search-enterprise/latest/support/index.md:14]: This should state Elasticsearch versions 7.10 - 7.17 inclusive. Not just 7.17

File: search-enterprise/latest/support/index.md [hunk 2]
```diff
@@ -11,7 +11,8 @@ The following are the supported platforms for Search Enterprise 3.1.x:
...
+| Opensearch server 1.3.2 | |
+| Alfresco Elasticsearch Connector 3.2.x | |
 
-> **Note:** The Elasticsearch server does not require any additional software from Alfresco in order to be used by Alfresco Search Enterprise 3.1
+> **Note:** The Elasticsearch/Opensearch server does not require any additional software from Alfresco in order to be used by Alfresco Search Enterprise 3.2
```
Comment (mjstanford) [inline search-enterprise/latest/support/index.md:18]: Slightly clunky. I would change to "Elasticsearch/Opensearch does not require any additional software from Alfresco in order to be used by Alfresco Search Enterprise 3.2"

**Review by mjstanford — CHANGES_REQUESTED** [review]

File: search-enterprise/latest/support/index.md (hunk 1 above)
Comment (oliveralfresco2018) [reply search-enterprise/latest/support/index.md:14]: This has been added to PR https://github.com/Alfresco/docs-alfresco/pull/839

File: search-enterprise/latest/support/index.md (hunk 2 above)
Comment (oliveralfresco2018) [reply search-enterprise/latest/support/index.md]: Changed

**Review by mjstanford — CHANGES_REQUESTED** [review]

File: search-enterprise/latest/support/index.md (hunk 1 above)
Comment (oliveralfresco2018) [reply search-enterprise/latest/support/index.md:14]: Thanks @mjstanford, yes that text is covered in the PR I mention. I am referring to this JIRA https://alfresco.atlassian.net/browse/DOCS-6945

**Review by mjstanford — APPROVED** [review]

**Pull request merged**